### PR TITLE
fix(logging): log connection config and charset info at startup

### PR DESCRIPTION
**Pull request by jrgm**
A little info of interest.

r? - @philbooth @rfk

### CODE REVIEW

File: lib/db/mysql.js [hunk 1]
```diff
@@ -950,5 +950,79 @@ module.exports = function (log, error) {
...
+      'character_set_client',
+      'character_set_connection',
+      'character_set_database',
+      'character_set_filesystem',
+      'character_set_results',
+      'character_set_server',
+      'character_set_system',
+      'collation_connection',
+      'collation_database',
+      'collation_server'
```
Comment (vbudhram) [inline lib/db/mysql.js]: nit: Possible to add `version` in here? We can also get this information from the AWS console, but I think having it here might be useful for debugging.

**Comment by vbudhram** [comment]
I like this! It could also be useful in the https://github.com/mozilla/fxa-oauth-server

File: lib/db/mysql.js (hunk 1 above)
Comment (jrgm) [inline lib/db/mysql.js]: Sure. Added `version`, and threw in `max_connections` and `wait_timeout` (because they haunt me from the past ;-) ).

**Comment by philbooth** [comment]
r+, looks good to me!

**Pull request merged**